tests/examplefiles/org/example.org

```diff
@@ -0,0 +1,145 @@
+# -*- org -*-
+#+title: Single Post with TOML front matter
+#+author:
+#+date: 2017-07-20
+#+options: creator:t toc:2
+
+#+hugo_base_dir: ../../
+#+hugo_categories: cat1 cat2
+#+hugo_menu: :menu "foo" :weight 10 :parent main :identifier single-toml
+#+description: Some description for this post.
+
+This is a single post. You do not need to set the =EXPORT_FILE_NAME=
+property in here. But then you also lose the tag and property
+inheritance Org awesomeness.
+
+line one \\
+line two
+
+*bold* /italics/ =verbatim= ~code~ +strikethrough+ _underline_
```

Comment:
The lexer trips up here: the test output does not contain the blank line (l18). It's possible that it's confusing `*bold*` with a heading line (`* heading`), but I am unsure how to fix this.

Reply:
This is now fixed!